launcher/screens/Guide/Dashboard/ConceptButton.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import { Image, StyleSheet, TouchableOpacity } from 'react-native';
+import Animated, {
+	useAnimatedStyle,
+	withTiming,
+} from 'react-native-reanimated';
+import { Hoverable, Text } from '@metacraft/ui';
+import { sharedStyle, useHoveredStyle } from 'screens/Guide/shared';
+import resources from 'utils/resources';
+
+interface Props {
+	label: string;
+	icon: number;
+	isFirst: boolean;
+	isLast: boolean;
+	onPress?: () => void;
+	isSelected: boolean;
+}
+
+const styles = StyleSheet.create({
+	container: {
+		alignItems: 'center',
+		justifyContent: 'center',
+	},
+	content: {
+		alignItems: 'center',
+	},
+	icon: {
+		width: 40,
+		aspectRatio: 1,
+	},
+	label: {
+		fontFamily: 'Poppins',
+		textAlign: 'center',
+		color: '#FFFBDF',
+	},
+});
+
+const ConceptButton: FC<Props> = ({
+	label,
+	icon,
+	isFirst,
+	isLast,
+	onPress,
+	isSelected,
+}) => {
+	const containerStyle = StyleSheet.flatten([
```

Comment:
@fluctlight-kayaba, is flatten necessary here? Could I use:
`const containerStyle = {
   ...styles.container,
  marginLeft: isFirst ? 0 : 20,
  marginRight: isLast ? 0 : 20,
}`